README.md

```diff
@@ -118,4 +118,26 @@ graph LR
     iadd-->enc_sa;
 
     softmax--"Probablity"-->pred;
-```
\ No newline at end of file
+```
+
+
+## The Folder Structure
+The folder of the project is desigened by slight structure for expansibility and convenience:
+```shell
+.
+├── data # mounted data and scripts
+│   ├── data_lodar.py
+│   └── nas
+├── eval.py # evaluation scipts
+├── models # transformer modules
+│   ├── decoder
+│   └── encoder
+├── README.md
+├── scripts #  scripts for convenience
+├── test.py # test interface
+├── train.py # train interface
```

Comment:
The comment does not seem necessary, consider remove.

Reply:
Done